network/src/connectivity_manager/mod.rs

```diff
@@ -597,19 +603,16 @@ where
                 self.dial_queue.remove(&peer_id);
             }
             peer_manager::ConnectionNotification::LostPeer(peer_id, addr, _origin, _reason) => {
-                match self.connected.get(&peer_id) {
-                    Some(curr_addr) if *curr_addr == addr => {
-                        // Remove node from connected peers list.
-                        self.connected.remove(&peer_id);
-                    }
-                    _ => {
-                        debug!(
-                            "{} Ignoring stale lost peer event for peer: {}, addr: {}",
-                            self.network_context,
-                            peer_id.short_str(),
-                            addr
-                        );
-                    }
+                if self.connected.contains_key(&peer_id) {
+                    // Remove node from connected peers list.
+                    self.connected.remove(&peer_id);
```

Comment:
do we want to log this as well?

Reply:
Luckily now we have the incoming / outgoing messages from the events.

I'll add a structured log for every event received

Reply:
Oh, turns out we already log the entire event in the structured logs event loop, with the internals, which would tell the directions and all.

Reply:
Agreed! But that doesn't show up per our previous conversation. Once that works, I think we can probably start eliminating a bunch of logs. I don't know if there's any value from having both structured and text logs. We eliminated text logs from the TCB unless there was something that didn't fit into the model for structured loggings.